File: Latihan3/Latihan3_Java/Processor.java
public class Processor {

	// class attribute
	String name;
	int price;

	// empty constructors
	Processor() {
	}

	// constructors
	Processor(String name, int price) {
		this.name = name;
		this.price = price;
	}

	// getter and setter
	public void setName(String name) {
		this.name = name;
	}

	public String getName() {
		return name;
	}

	public void setPrice(int price) {
		this.price = price;
	}

	public int getPrice() {
		return price;
	}

}

File: Latihan3/Latihan3_Python/Processor.py
class Processor:
    # class attribute
    __name = ""
    __price = 0

    # constructors
    def __init__(self, name="", price=0):
        self.__name = name
        self.__price = price

    # getter and setter
    def setName(self, name):
        self.__name = name

    def getName(self):
        return self.__name

    def SetPrice(self, price):
        self.__price = price

    def getPrice(self):
        return self.__price
